pkgs/top-level/perl-packages.nix

```diff
@@ -16630,6 +16630,10 @@ let
       description = "Indent and reformat perl scripts";
       license = lib.licenses.gpl2Plus;
     };
+    nativeBuildInputs = lib.optional stdenv.isDarwin shortenPerlShebang;
+    postInstall = lib.optionalString stdenv.isDarwin ''
+      shortenPerlShebang $out/bin/perltidy
```

Comment:
Is there any harm to do this on all platforms? It would make it more straight forward.